src/main/java/net/pms/util/FileUtil.java

```diff
@@ -546,6 +547,10 @@ public static String getFileNamePrettified(String f, File file, DLNAMediaInfo me
 
 		formattedName = basicPrettify(filename);
 
+		if (formattedName.toLowerCase().endsWith("sample")) {
```

Comment:
You should always use ```Locale.ROOT``` when using locale sensitive functions like `` toLowerCase()``` unless you actually wants it to do it in a locale-sensitive way. When you don't specify it, it will use the OS default locale, which means that the results are unpredictable.

Reply:
@Nadahar I have added a change to use English "locale" because it is only doing lowercase to compare with an English string `sample`. If that is a good fix can you approve and merge, or if you think `Locale.ROOT` is better please explain or just make the change

Reply:
@SubJunk ```Locale.English``` is perfectly fine AFAIK, the reason I use ```Locale.ROOT``` is because I understand it as that the "root" locale simply means "no special handling". I understand it as if the "root" locale is the "default"/basis/fallback for all the others, and that the other locales then basically just implements the deviations. When it comes to case conversion I'm pretty sure there's no "special handling" for English, so it will probably end up running the exact same code. I simply explicitly use ```Locale.ROOT``` to signal that I don't want any localized handling.

AFAICR Android doesn't have ```Locale.ROOT``` and recommends using ```Locale.English``` instead, so it's really just semantics.

When it comes to approving it, I guess I could since it's a small change and you probably knows this code well, but I haven't really studied what it does and would prefer if @valib could just update his review.

I understand the idea behind preserving the "sample" so you can tell the difference from "the real thing", but I'm not quite sure of how this relates to the "edition".

Reply:
Cool, thanks for the reply. Yeah it is a bit of a hack to put it onto the end of the edition instead of making another parameter like `isSample` to `return`